will/acl.py

```diff
@@ -20,18 +20,18 @@ def get_acl_members(acl):
     return acl_members
 
 
-def is_acl_allowed(nick, acl):
+def is_acl_allowed(_id, acl):
     if not getattr(settings, "ACL", None):
         logging.warning(
             "%s was just allowed to perform actions in %s because no ACL settings exist. This can be a security risk." % (
-                nick,
+                _id,
                 acl,
             )
         )
         return True
     for a in acl:
         acl_members = get_acl_members(a)
-        if nick in acl_members or nick.lower() in [x.lower() for x in acl_members]:
+        if _id in acl_members:
```

Comment:
single leading underscore usually denotes some kind of private class variable. In this case the best practice is to add a single trailing underscore: `id_`

Reply:
since this was not modified yet. what's your opinion on giving to this variable a more understandable name? maybe something like `user_id`?

Reply:
Yep that's a fair point too! In the future @wontonst if you have good ideas like this about not just what ought to be made better but thoughts on specifically how it could be improved you don't need to be coy. I'm not delicate, promise! Just throw 'em all in one comment and save me a trip :)